src/js/themes/base.js

```diff
@@ -1259,6 +1259,56 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         },
       },
     },
+    pageHeader: {
+      actions: {
+        // any box props
+        align: 'end',
+      },
+      context: {
+        // any box props
+        align: 'start',
+      },
+      subtitle: {
+        // any paragraph props
+        margin: 'none',
+        fill: true,
+      },
+      title: {
+        // any heading props
+        margin: 'none',
+        fill: true,
+      },
+      small: {
```

Comment:
Still waffling back and forth if we should a) group these breakpoints under a single property, b) keep them at this level, or c) or restructure to read component.property[breakpoint]? 

Looking through the rest of the theme file, I think I am seeing "c" most often.

@ericsoderberghp any thoughts? Preferences?

Reply:
We have been a bit inconsistent about this in the theme in the past. Sometimes we rooted more in aspect, with state variations beneath the aspect, as with `heading.level.1.size` or `tab.border.hover`. Other times, we rooted more in state, as with `button.hover.primary.background`. Our current direction is to go with the later, rooting more in state than in aspect. So for this, we'd do "b".
Admittedly, our theme patterns are bit of a mess as theming has grown.